### ISSUE TITLE
Add support for dashes in tags in match_specification

### ISSUE THREAD

**Issue by steinex** [gessnerfl/terraform-provider-instana#12]
Hi!

We added the following patch to `instana/filterexpression/filter-expression-parser.go` to support dashes for tags like `call.http.header.x-example-foo` in match_specification.

```
@@ -109,7 +109,7 @@ func (e *UnaryOperationExpression) Render() string {
 var (
        filterLexer = lexer.Must(lexer.Regexp(`(\s+)` +
                `|(?P<Keyword>(?i)OR|AND|TRUE|FALSE|IS_EMPTY|NOT_EMPTY|IS_BLANK|NOT_BLANK|EQUALS|NOT_EQUAL|CONTAINS|NOT_CONTAIN)` +
-               `|(?P<Ident>[a-zA-Z_][\.a-zA-Z0-9_]*)` +
+               `|(?P<Ident>[a-zA-Z_][\.a-zA-Z0-9_-]*)` +
                `|(?P<Number>[-+]?\d+(\.\d+)?)` +
                `|(?P<String>'[^']*'|"[^"]*")` +
                `|(?P<Operators>EQUALS|NOT_EQUAL|CONTAINS|NOT_CONTAIN|IS_EMPTY|NOT_EMPTY|IS_BLANK|NOT_BLANK)`,
```

We would have sent a pull request, but we weren't able to fix your tests. It would be nice to have support for this added officially.

**Comment by gessnerfl** [comment]
@steinex this makes a lot of sense. The issue with your proposed change is that it is a regex and a dash needs to be escaped.
However I already provided a fix and will try to provide a fix soon

**Comment by gessnerfl** [comment]
Implemented with [version 0.2.1](https://github.com/gessnerfl/terraform-provider-instana/releases/tag/v0.2.1)
